recipes-extended/xen/xen-common.inc

```diff
@@ -47,7 +47,6 @@ SRC_URI_append = " \
     file://libxl-pci-passthrough-fixes.patch \
     file://libxl-vwif-support.patch \
     file://libxl-atapi-pt.patch \
-    file://libxl-iso-hotswap.patch \
```

Comment:
Did you have the async friendly way already mapped out or is that something we'll have to implement?

Reply:
ack ty.